### DISCUSSION TITLE
exe file works in Anaconda Prompt window but not in Windows command line

### DISCUSSION THREAD

**Discussion by JuJaFlash** [pyinstaller/pyinstaller#5483] [Help] [answered]
Hello everyone,

My python code include tkinter and matplotlib. I create a .exe with pyinstaller. 
When I run it with Anaconda Prompt, the message appears and my interface run normally. If I run it with Windows command line (or double clik) the message appears but no interface appears.

If I suppress matplotlib import, the exe works with Windows command line too. The issue is clearly caused by matplotlib, but I have no idea how to solve it. Can you help me ?

Thank you.

Using: Python 3.7.9 Pyinstaller 3.6 Matplotlib 3.2.2

**Comment by bwoodsend** [comment, marked as answer]
That'll be [this miserable DLL](https://github.com/pyinstaller/pyinstaller/issues/4968#issuecomment-662075970) again.

**Reply by JuJaFlash** [reply]
It works ! Thank you so much !
